LadybugTools_Engine/Python/src/ladybugtools_toolkit/external_comfort/_externalcomfortbase.py

```diff
@@ -326,6 +328,59 @@ def plot_utci_heatmap(
             title=self.description(),
         )
 
+    def walkability_time_limits(self):
+
+        """Calculate walkability time limiits
+        Returns:
+            HourlyContinuousCollection: An object containing walkability values for each hour. 
+        """
+        csv_file = r"C:\ProgramData\BHoM\Extensions\PythonCode\LadybugTools_Toolkit\src\data\walkability.csv"
+        df = pd.read_csv(csv_file)
+        walkability_vals = []
+
+        for utci_val in self.universal_thermal_climate_index.values:
+            utci_int = int(round(utci_val,0))
+            utci_str = str(utci_int)
+
+            val = df.loc[df["UTCI temperature"] == utci_str, 'Time within no thermal stress and moderate thermal stress bands (mins)'].values
+            if utci_int < 20:
+                val = df.loc[df["UTCI temperature"] == "defaultlower", 'Time within no thermal stress and moderate thermal stress bands (mins)'].values
+            elif utci_int > 60:
+                val = df.loc[df["UTCI temperature"] == "defaultupper", 'Time within no thermal stress and moderate thermal stress bands (mins)'].values
+
+            walkability_vals.append(val[0])
+
+        header = header_from_string("Walkability Time (minutes)")
+
+        return HourlyContinuousCollection(header=header, values=walkability_vals)
+
+    def plot_walkability_heatmap( 
+            self,
+            ax: plt.Axes = None,
+        ) -> plt.Axes:
+            
+        """Create a walkability heatmap showing 
+        Args:
+            ax (plt.Axes, optional): A matplotlib Axes object to plot on. Defaults to None.
+            utci_categories (Categorical, optional): The UTCI categories to use. Defaults to
+                UTCI_DEFAULT_CATEGORIES.
+        Returns:
+            plt.Axes: A matplotlib Axes object.
+        """
+
+        cmap_name = "Walkability colours"
+        cmap_colours = ["red","yellow","white"]
+        colourmap = LinearSegmentedColormap.from_list(cmap_name, cmap_colours, N=100)
+
+        return heatmap(
+            series = collection_to_series(self.walkability_time_limits()),
+            ax=ax,
+            cmap = colourmap,
+            vmin=0,
+            vmax=15,
+            title=self.description()
+        )
```

Comment:
```suggestion
        return heatmap(
            series = collection_to_series(self.walkability_time_limits()),
            ax=ax,
            cmap = colourmap,
            vmin=0,
            vmax=15,
            title=kwargs.pop("title", self.description()),
            **kwargs
        )
```
this will allow the title to be modified by a user if they add the key-word arg "title", and any other kwargs will be passed through to the heatmap function and handled there :)